cms/envs/common.py

```diff
@@ -1264,7 +1264,7 @@
 
 ############## Settings for Studio Context Sensitive Help ##############
 
-HELP_TOKENS_INI_FILE = REPO_ROOT / "docs" / "cms_config.ini"
+HELP_TOKENS_INI_FILE = REPO_ROOT / "cms" / "envs" / "help_tokens.ini"
```

Comment:
Unless there are references outside of the repo, this is all I found.  I thought these names were more clear, but let me know.  I know I dropped "cms" and "lms" from the names, but that is because they moved to the `cms/envs` and `lms/envs` directories, which are use the directory name for namespacing.